webserver/templates/navbar.html

```diff
@@ -28,7 +28,7 @@
       {# Not showing user stuff on error pages. If attempt to load user info fails there, page will not render. #}
       {% if not error %}
         <ul class="nav navbar-nav navbar-right">
-          {% if not current_user or current_user.is_anonymous() %}
+          {% if not current_user or current_user.is_anonymous %}
```

Comment:
yeah, I upgraded flask login, so I guess that's it.

django's templating language will take a name and first try it as a property, then a dictionary key, then a function. does jinja do the same?